elements/pfelement/src/pfelement.js

```diff
@@ -277,6 +279,20 @@ class PFElement extends HTMLElement {
   log(...msgs) {
     PFElement.log(`[${this.tag}]`, ...msgs);
   }
+
+  emitEvent(
+    name,
+    { bubbles = true, cancelable = false, composed = false, detail = {} } = {}
```

Comment:
```suggestion
    { bubbles = true, cancelable = false, composed = false, detail = {} } = {},
    tag = this.tag
```

Can we set the tag as a default and allow overrides optionally?

Reply:
It's definitely possible but I this the current approach is more straightforward.  One argument for the current approach is that the `events` object contains the names of the events.  With the suggested approach, `events` would no longer be as useful because the true event names wouldn't be created until `emitEvent` is called.  Outside code would be forced to hardcode event names.  Not the end of the world, but I would like to offer the option to `import PfeAccordion` and then `addEventListener(PfeAccordion.events.open, ...)` in order to tap into PfeAccordion's event names.

An argument against the current approach is that it leaves event naming conventions up to the developer to implement and diligent reviewers to ensure.  I'm usually the one arguing for automation over diligence! :stuck_out_tongue:  But in this case, I really like having the `events` object for quick lookup of event names.

Reply:
👍